app/assets/javascripts/directives/task_item.js

```diff
@@ -0,0 +1,55 @@
+angular.module("OpenHq").directive("taskItem", function(Task) {
+  return {
+    restrict: "E",
+    scope: {
+      task: '=',
+      users: '=',
+    },
+    template: JST['templates/directives/task_item'],
+
+    controller: function($scope){
+      $scope.editing = false;
+
+      $scope.startEditing = function($event) {
+        $scope.editing = true;
+        $($event.currentTarget).closest('li').addClass('editing');
```

Comment:
oh nevermind seen that. Should set the class name by the scope then

`<li ng-class="{editing: editing}">`